LFS171x/sawtooth-material/sawtooth-tuna/processor/handlers.js

```diff
@@ -0,0 +1,140 @@
+'use strict'
```

Comment:
Provide license information for this file.
`// SPDX-License-Identifier: Apache-2.0`